pages/specification/migration/_index.md

```diff
@@ -5,76 +5,76 @@ section: docs
+| `$dynamicRef`           | Core          | 2020-12            | No      | Replaced `$recursiveRef`       |
+| `$recursiveAnchor`      | Core          | 2019-09            | No      | Replaced by `$dynamicAnchor`   |
+| `$recursiveRef`         | Core          | 2019-09            | No      | Replaced by `$dynamicRef`      |
+| `$ref`                  | Core          | 2019-09            | No      | No                             |
+| `$schema`               | Core          | 02                 | No      | No                             |
+| `$vocabualry`           | Core          | 2019-09            | No      | No                             |
+| `additionalItems`       | Core          | 03                 | No      | No                             |
+| `allOf`                 | Core          | 04                 | No      | No                             |
+| `anyOf`                 | Core          | 04                 | No      | No                             |
+| `const`                 | Validation    | 06                 | No      | No                             |
+| `contains`              | Core          | 06                 | No      | No                             |
+| `contentEncoding`       | Validation    | 07                 | No      | No                             |
+| `contentMediaType`      | Validation    | 07                 | No      | No                             |
+| `contentSchema`         | Validation    | 2019-09            | No      | No                             |
+| `definitions`           | Validation    | 04                 | No      | Replaced by` $defs`            |
+| `default`               | Validation    | 01                 | No      | No                             |
+| `dependencies`          | Core          | 03                 | No      | No                             |
+| `dependentRequired`     | Validation    | 2019-09            | No      | No                             |
+| `dependentSchemas`      | Core          | 2019-09            | No      | Replaced `dependencies`        |
+| `deprecated`            | Validation    | 2019-09            | No      | No                             |
+| `disallow`              | Validation    | 03                 | Yes     | No                             |
+| `divisibleBy`           | Validation    | 02                 | No      | Replaced by `multipleOf`       |
+| `else`                  | Core          | 07                 | No      | No                             |
+| `enum`                  | Validation    | 01                 | No      | No                             |
+| `examples`              | Validation    | 06                 | No      | No                             |
+| `exclusiveMaximum`      | Validation    | 03                 | No      | Replaced `maximumCanEqual`     |
+| `exclusiveMinimum`      | Validation    | 03                 | No      | Replaced `minimumCanEqual`     |
```

Comment:
This is another case where the keyword changed behavior without changing its name.

```suggestion
| `exclusiveMaximum` (legacy) | Validation    | 03                 | 06      | Replaced `maximumCanEqual`           |
| `exclusiveMaximum`          | Validation    | 06                 | No      | Replaced `exclusiveMaximum` (legacy) |
| `exclusiveMinimum` (legacy) | Validation    | 03                 | 06      | Replaced `minimumCanEqual`           |
| `exclusiveMinimum`          | Validation    | 06                 | No      | Replaced `exclusiveMinimum` (legacy) |
```